### PR TITLE
Last batch of misc cleanup

### PR DESCRIPTION
**Pull request by KrzysztofCwalina**
cc: @ahsonkhan, @GrabYourPitchforks, @joshfree, @A-And

### CODE REVIEW

File: src/System.Text.Utf8String/System/Text/Utf8String.cs [hunk 1]
```diff
@@ -12,22 +12,22 @@
 namespace System.Text.Utf8
 {
     [DebuggerDisplay("{ToString()}u8")]
-    public class Utf8String
+    public sealed class Utf8String
```
Comment (ahsonkhan) [inline src/System.Text.Utf8String/System/Text/Utf8String.cs]: What motivated the decision to seal this class?

**Comment by ahsonkhan** [comment]
@dotnet-bot test this please

**Comment by KrzysztofCwalina** [comment]
@dotnet-bot test this

File: src/System.Text.Utf8String/System/Text/Utf8String.cs (hunk 1 above)
Comment (KrzysztofCwalina) [reply src/System.Text.Utf8String/System/Text/Utf8String.cs]: System.String is sealed. Also, if/once we change this type to special runtime type (variable size class), inheriting from this type won't be possible.

**Pull request merged**